softvis3d-frontend/package.json

```diff
@@ -64,6 +64,8 @@
     "css-loader": "^0.26.1",
     "enzyme": "^2.7.0",
     "extract-text-webpack-plugin": "^1.0.1",
+    "jsdom": "9.10.0",
```

Comment:
For what do we need jsdom? I see its referenced in the mocha-opts, but i can not see any usages of it within the tests.